### PR TITLE
✨(Feature, UI, API, Tests) : Created Explore Feature

### PR DESCRIPTION
**Pull request by PxttytheThxtty**

### CODE REVIEW

File: libs/au-pair-profile/feature/src/lib/au-pair-profile.component.html [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
-    <div class = "parent-profile-div">
-        <div class = 'title-div'>
-            <p class='profile-name'>{{this.userDetails.fname}}</p>
+    <div class = "profile-div">    
+        <h1 class='profile-name'>{{this.userDetails.fname}} {{this.userDetails.sname}}</h1>
+        <div class='profile-picture-wrapper'>
+            <img class="profile-picture" src="assets/images/placeholder-profile.jpg" alt="">
         </div>
-        <!-- <div class = "profile-picture-div">
-            <div class='profile-picture-wrapper'>
```
Comment (Ruben-Brits) [inline libs/au-pair-profile/feature/src/lib/au-pair-profile.component.html:8]: can this be removed?

File: libs/au-pair-profile/feature/src/lib/au-pair-profile.component.scss [hunk 2]
```diff
@@ -1,3 +1,13 @@
+.profile-div{
+    width: 90%;
+    margin-top: 60px;
+    margin-left: auto;
+    margin-right: auto;
+    background-color: #69A88DB2;
```
Comment (Ruben-Brits) [inline libs/au-pair-profile/feature/src/lib/au-pair-profile.component.scss:6]: please use var(--ion etc etc)

File: libs/edit-parent-profile/feature/src/lib/edit-parent-profile.component.scss [hunk 3]
```diff
@@ -33,71 +47,26 @@
...
-
-.parent-info-small{
+.details-div{
     height: 6%;
     background-color: var(--ion-background-color);
     display:flex;
     font-family: Roboto;
     align-items: center;
-    color: #79BA9E;
+    color: black;
```
Comment (Ruben-Brits) [inline libs/edit-parent-profile/feature/src/lib/edit-parent-profile.component.scss:56]: i think we use --ion-color-dark not black, unless this had specific reason to black

File: libs/explore/feature/src/lib/expand-modal/expand-modal.component.scss [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+    font-family: Roboto;
+    align-items: center;
+    color: black;
+    margin-bottom: 12px;
+    box-shadow: 1px 2px 5px rgba(0, 0, 0, 0.13);
+    border-radius: 6px;
+}
+
+.profile-picture-wrapper{
+    width: 166px;
```
Comment (Ruben-Brits) [inline libs/explore/feature/src/lib/expand-modal/expand-modal.component.scss:54]: does this hardcoded with work with scaling?

File: libs/explore/feature/src/lib/explore.component.html [hunk 5]
```diff
@@ -0,0 +1,86 @@
+<the-au-pair-navbar></the-au-pair-navbar>
+<ion-content [fullscreen]="true">
+    
+
```
Comment (Ruben-Brits) [inline libs/explore/feature/src/lib/explore.component.html:4]: spacing

File: libs/explore/feature/src/lib/explore.component.html [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        </div>
+        <div *ngIf="AuPairArray.length === 0" class="noaupair" id="page-wrapper">
+            There are no Au Pairs matching your filters
+        </div>
+    </div>
+
+    <ion-menu content-id="page-wrapper" class="menu">
+        <ion-header class="menu-header">
+                <ion-toolbar translucent>
+                    <ion-title></ion-title>
```
Comment (Ruben-Brits) [inline libs/explore/feature/src/lib/explore.component.html:52]: empty titlte

File: libs/explore/feature/src/lib/explore.component.spec.ts [hunk 7]
```diff
@@ -0,0 +1,30 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
```
Comment (Ruben-Brits) [inline libs/explore/feature/src/lib/explore.component.spec.ts]: empty line

File: libs/explore/feature/src/lib/explore.component.ts [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+      error=>{console.log("Error has occured with API: " + error);}
+    )
+  }
+
+  async setAuPairArray()
+  {
+    this.auPairs.forEach((ap: { id: any; rating: any; payRate: any; fname: any; sname: any, suburb: any; employer: any; age: any; gender: any; longitude: any; latitude: any; distance: any;}) => {
+      this.serv.getUser(ap.id).subscribe(
+        res=>{
+
```
Comment (Ruben-Brits) [inline libs/explore/feature/src/lib/explore.component.ts:89]: empty line

File: libs/parent-profile/feature/src/lib/parent-profile.component.html [hunk 9]
```diff
@@ -1,15 +1,10 @@
...
 <ion-content [fullscreen]="true">
     <div class = "parent-profile-div">
-        <div class = 'title-div'>
-            <p class='profile-name'>{{this.userDetails.fname}}</p>
-        </div>
-        <!-- <div class = "profile-picture-div">
+        <h1 class='profile-name'>{{this.userDetails.fname}} {{this.userDetails.sname}}</h1>
             <div class='profile-picture-wrapper'>
-                <ion-icon name="people-circle-outline" class='profile-picture'></ion-icon>
+                <img class="profile-picture" src="assets/images/placeholder-profile.jpg" alt="">
```
Comment (Ruben-Brits) [inline libs/parent-profile/feature/src/lib/parent-profile.component.html:6]: empty alt

File: libs/parent-profile/feature/src/lib/parent-profile.component.ts [hunk 10]
```diff
@@ -49,7 +54,7 @@ export class ParentProfileComponent implements OnInit {
   async getUserDetails()
   {
     /* User Details */
-    await this.serv.getUser("4561237814867").subscribe(
+    await this.serv.getUser("0105845069834").subscribe(
```
Comment (Ruben-Brits) [inline libs/parent-profile/feature/src/lib/parent-profile.component.ts:57]: please make note to update this hardcoded stuff to use store.

File: libs/parent-profile/feature/src/lib/parent-profile.component.ts [hunk 11]
```diff
@@ -60,10 +65,15 @@ export class ParentProfileComponent implements OnInit {
...
+        this.userDetails.latitude = res.latitude;
+        this.userDetails.longitude = res.longitude;
+        this.userDetails.suburb = res.suburb;
+        this.userDetails.gender = res.gender;
+        this.userDetails.age = res.age;
       },
       error=>{console.log("Error has occured with API: " + error);}
     )
-      await this.serv.getParent("4561237814867").subscribe(
+      await this.serv.getParent("0105845069834").subscribe(
```
Comment (Ruben-Brits) [inline libs/parent-profile/feature/src/lib/parent-profile.component.ts:76]: same here

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]

File: libs/explore/feature/src/lib/explore.component.scss [hunk 12]
```diff
@@ -0,0 +1,278 @@
...
+.au-pair-card-div-wrapper {
+    text-align: center;
+    width: 100%;
+    position: relative;
+    height: 100%;
+    display: block;
+    margin-top: 47px;
+    margin-left: auto;
+    margin-right: auto;
+    // z-index: -1;
```
Comment (ricecakesreece) [inline libs/explore/feature/src/lib/explore.component.scss:130]: commented out code

File: libs/explore/feature/src/lib/explore.component.scss [hunk 13]
```diff
@@ -0,0 +1,278 @@
...
+    width: 140px;
+    height: 140px;
+    border-radius: 50%;
+}
+
+.au-pair-name{
+    font-family: 'TitanOne';
+    margin-top: 7.5px;
+    color: var(--ion-color-dark);
+    // width: 100%;
```
Comment (ricecakesreece) [inline libs/explore/feature/src/lib/explore.component.scss:193]: commented out

**Review by ricecakesreece — APPROVED** [review]

**Review by Ruben-Brits — APPROVED** [review]

**Pull request merged**